plugins/providers/metabase/client.go

```diff
@@ -129,7 +131,7 @@ func NewClient(config *ClientConfig) (*client, error) {
 }
 
 func (c *client) GetDatabases() ([]*Database, error) {
-	req, err := c.newRequest(http.MethodGet, "/api/database", nil)
+	req, err := c.newRequest(http.MethodGet, "/api/database?include=tables", nil)
```

Comment:
why do we need tables here?

Reply:
this one filter will fetch tables in the same fetch database API call